7-ExecutablesAndLibraries/README.md

```diff
@@ -1 +1,42 @@
-# Executables and libraries
\ No newline at end of file
+# Executables and libraries
+
+This chapter explains how to create executables and libraries. Executables and libraries are essential for managing projects
+with CMake.
+
+Executables and libraries are targets. A target is (just as in a Makefile) something that is going to be built, and which can be a required dependency
+for another target.
+
+# Executable
+
+An executable target is used to create for example a `.exe` file or for cross platform a `.elf`, `.hex`, or a `.elf`file.
```

Comment:
An executable target generates files such as  `.exe` for Windows or cross-platform formats like `.bin`, `.elf` or `.hex`.

Reply:
Fixed it